stdlib/Project.pkl

```diff
@@ -91,18 +91,15 @@ package: Package?
 /// ```
 tests: Listing<String>(isDistinct)
 
-/// Tells if the project is a file-based module named `PklProject`.
-local isLocalPklProject = (it: Project) -> 
-  it.projectFileUri.startsWith("file:") && it.projectFileUri.endsWith("/PklProject")
+/// Tells if the project is a local module named `PklProject`, is not self, and has a [package] section
+local isValidLoadDependency = (it: Project) ->
+  isUriLocal(projectFileUri, it.projectFileUri)
+  && it.projectFileUri.endsWith("/PklProject")
+  && it != module
+  && it.package != null
 
-/// A local dependency is another [Project] that is local to the file system.
-///
-/// To declare, use `import("path/to/PklProject")`
-typealias LocalDependency = Project(
-  isLocalPklProject,
-  this != module,
-  this.package != null
-)
+const local function isUriLocal(uri1: String, uri2: String): Boolean =
+  uri1.substring(0, uri1.indexOf(":")) == uri2.substring(0, uri2.indexOf(":"))
```

Comment:
This isn't the perfect check. This should also check that the URIs have the same authority. This is hard to implement without a URI API within the stdlib though.